.gitmodules

```diff
@@ -9,7 +9,8 @@
 	url = https://github.com/tonymillion/Reachability.git
 [submodule "spec/cocoapods-integration-specs"]
 	path = spec/cocoapods-integration-specs
-	url = https://github.com/CocoaPods/cocoapods-integration-specs.git
+	url = https://github.com/philippelatulippe/cocoapods-integration-specs.git
```

Comment:
we do not want to change this as it will point to your fork. I am going to merge your integration specs. I will merge your integration specs PR and then simply bump the SHA to the submodule here.